server/views/serviceProviderReferrals/interventionProgress.njk

```diff
@@ -10,67 +10,131 @@
 
 {% block referralPageSection %}
 
-  {% if presenter.referralEnded %}
-     {{ govukNotificationBanner(cancelledReferralNotificationBannerArgs) }}
-  {% endif %}
-
-  {% if presenter.referralAssigned %}
-    <p class="govuk-body">This intervention is assigned to <strong>{{ presenter.assignedCaseworkerFullName }}</strong> (<a href="mailto:{{ presenter.assignedCaseworkerEmail }}">{{ presenter.assignedCaseworkerEmail }}</a>).</p>
-  {% endif %}
-
-  <h2 class="govuk-heading-m">Supplier assessment appointment</h2>
-
-  {% if presenter.referralAssigned %}
-    <p class="govuk-body">
-     {{ supplierAssessmentMessage }}
-    </p>
-
-    {{ govukTable(supplierAssessmentAppointmentsTableArgs(govukTag)) }}
-  {% else %}
-    <p class="govuk-body">
-      Once a caseworker has been assigned the initial assessment can be booked.
-    </p>
-    <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
-  {% endif %}
-
-  <h2 class="govuk-heading-m">Action plan</h2>
-
-  {% if presenter.referralAssigned %}
-    <p class="govuk-body">
-      Complete the action plan and submit it to the probation practitioner for approval.
-    </p>
-
-    {{ govukTable(actionPlanTableArgs(govukTag, csrfToken)) }}
-
-  {% else %}
-    <p class="govuk-body">
-      Once a caseworker has been assigned the action plan can be sumbitted.
-    </p>
-    <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
-
-  {% endif %}
-
-  <h2 class="govuk-heading-m">Session progress</h2>
-  <p class="govuk-body">
-    Track the progress of each intervention session.
-  </p>
-
-  {% if presenter.hasSessions %}
-    {{ govukTable(sessionTableArgs(govukTag)) }}
-  {% else %}
-    <p class="govuk-body">
-      Sessions will appear here when the action plan is approved.
-    </p>
-
-    <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
-  {% endif %}
-
-  <h2 class="govuk-heading-m">End of service report</h2>
-
-  <p class="govuk-body">
-  Please complete the end of service report once the intervention has ended.
-  </p>
-
-  {{ govukSummaryList(endOfServiceReportSummaryListArgs(govukTag, csrfToken)) }}
```

Comment:
can we change to 2 space indents instead of tabs, to keep the formatting consistent with the js code